vertical-pod-autoscaler/deploy/vpa-rbac.yaml

```diff
@@ -57,6 +57,19 @@ rules:
 ---
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
+metadata:
+  name: system:vpa-status-actor
+rules:
+  - apiGroups:
+      - "autoscaling.k8s.io"
+    resources:
+      - verticalpodautoscalers/status
+    verbs:
+      - get
+      - patch
```

Comment:
One thing I missed in the first review: Now that we have a `/status` subresource, the `system:vpa-actor` no longer needs permission to `patch` the VPA objects, so we can also remove that as part of this change.

Reply:
Line 47 still gives `patch` permission to role `system:vpa-actor`

Reply:
done